sdk/src/builder.rs

```diff
@@ -3542,6 +3542,8 @@ mod tests {
     #[test]
     fn test_builder_set_base_path() {
         let mut builder = Builder::new();
+        builder.set_intent(BuilderIntent::Create(DigitalSourceType::Empty));
```

Comment:
The ingredient/ingredient.json file lists the ingredient as componentOf, so this is the correct intent here,